code/controllers/subsystem/rogue/treasury.dm

```diff
@@ -80,7 +80,7 @@ SUBSYSTEM_DEF(treasury)
 		var/people_told = 0
 		for(var/mob/living/carbon/human/X in GLOB.human_list)
 			switch(X.job)
-				if("King", "Steward", "Clerk")
+				if("Lord", "Steward", "Clerk")
```

Comment:
Feels like these should be comparing typepaths rather than strings, are there typepaths for this? But regardless, this is good.